src/docs/development/tools/devtools/visual-debugging.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Visual debugging
+description: A guide to Flutter's visual debugging functionality.
+---
+
+The Flutter Inspector provides several options for visually debugging your app.
+These are the options available from the inspector within the DevTools.
+
+## Slow Animations
+
+When enabled, this option runs animations 5 times slower for easier visual
+inspection.
+This can be useful if you want to carefully observe and tweak an animation that
+doesn’t look quite right.
+
+This can also be set in code:
+
+```dart
+import 'package:flutter/scheduler.dart' 
+
+timeDilation = 5.0;
+```
+ 
+This will slow the animations by 5x.
+
+### See also
+
+* [Flutter documentation: timeDilation property](https://api.flutter.dev/flutter/scheduler/timeDilation.html) 
+
+### Example
+
+![Screen recording showing normal animation speed]({% asset tools/devtools/debug-toggle-slow-animations-disabled.gif @path %})
+![Screen recording showing slowed animation speed]({% asset tools/devtools/debug-toggle-slow-animations-enabled.gif @path %})
+
+## Show Guidelines
+
+This option draws guidelines on your app which show render boxes, alignments,
+paddings, scroll views, clippings and spacers.
+
+This tool can be used for better understanding your layout, for instance
+finding unwanted paddings or understanding how widgets are being aligned.
+
+You can also enable this in code:
```

Comment:
- I added "layout" to the header, otherwise it sounds like a guide for a performance.
- 
```suggestion
## Show layout guidelines

This feature draws guidelines over your app that displays render boxes, alignments,
paddings, scroll views, clippings and spacers.

This tool can be used for better understanding your layout, for instance
finding unwanted paddings or understanding how widgets are being aligned.

You can also enable this in code:
```